src/components/Card/Card.tsx

```diff
@@ -0,0 +1,218 @@
+import React, { FC, Ref, useContext } from 'react';
+import { CardProps, CardSize, CardType } from './Card.types';
+import { mergeClasses } from '../../shared/utilities';
+import { ButtonShape, TwoStateButton } from '../Button';
+import { SizeContext, Size } from '../ConfigProvider';
+import { Icon, IconName, IconSize } from '../Icon';
+import { Stack } from '../Stack';
+import styles from './card.module.scss';
```

Comment:
nit: move styles as the last import with a line separating it, makes it easy to see this component uses a module.